anchor-reference-server/src/test/kotlin/org/stellar/anchor/reference/service/UniqueAddressServiceTest.java

```diff
@@ -0,0 +1,4 @@
+package org.stellar.anchor.reference.service;
```

Comment:
The test is empty and can be removed.

I am not sure if we want to add unit tests to the reference server at this stage.

Reply:
Thanks for surfacing that. FIxed in https://github.com/stellar/java-stellar-anchor-sdk/pull/440/commits/4e366835094714b123ee2a7b28198d00675bc3d3.